ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/ansible_eos_device.py

```diff
@@ -27,6 +27,14 @@
     HAS_ANTA = False
     # Next line to make ansible-test sanity happy
     AntaDevice = object
+except TypeError as e:
+    # Known bug with Python 3.9.7 and Pydantic `conint`, impacting ANTA. Issue: https://github.com/arista-netdevops-community/anta/issues/557
+    if "Interval() takes no arguments" in str(e):
+        msg = "Encountered a known bug with Pydantic and Python 3.9.7. Please consider upgrading Python or using a different environment."
```

Comment:
Users of AVD do not know anything about ANTA / Pydantic so the message may need to be more explicit

Something like

The ANTA testing framework leveraged in eos_validate_state is using a libary that is unable to load for some python 3.9.x versions (to the best of our knowledge).

Or something better said but we don't have a lot of info :). and probably invite them to a discussion on AVD / ANTA to report their python version where it failed :)

Reply:
Updated in 359bec3b2eed78e105c6eb215a979e7a773bc325